tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -55,6 +55,12 @@ class GivenUsingSuite extends BaseDottySuite {
     )
   }
 
+  test("given-anon-as") {
```

Comment:
So this one currently works in Dotty, reported it as weird here https://github.com/lampepfl/dotty/issues/10074

Reply:
Added!